_editorial_policies/02_editor_instructions.md

```diff
@@ -11,28 +11,32 @@ This provides instructions to editors handling manuscripts for LiveCoMS.
+Once the presubmission letter is approved, the author proceeds with article preparation and ultimately submits the article to LiveCoMS.
+The Lead Editor then checks that the article is consistent with what was proposed in the presubmission letter, and assigns it to an appropriate Associate Editor who manages the review process and makes a final decision on the manuscript, reporting the final decision to the authors. Usually this Associate Editor will be the same one that handled the original presubmission letters.
+
+## Pre-submission letters
+
+Assuming the subject matter proposed in the presubmission letter is within the scope of LiveCoMS and is otherwise acceptable, the Section Lead Editor assigns this presubmission letter to an Associate Editor. Reasons for the letter not being acceptable include, for example, ignoring key required elements of the presubmission letter or being too similar to existing LiveCoMS articles. The Section Lead Editor signs off on the final presubmission letter, and the Associate Editor notifies the author that the letter is approved.  Instructions for the presubmission letter are [found here](https://github.com/livecomsjournal/livecomsjournal.github.io).
 
 ## Pre-review processing
 
 ### Lead Editor
 
-The Lead Editor:
-- Handles analysis of presubmission inquiries
-- On article submission, confirms that the article is consistent with the presubmission letter
-- Passes the article and presubmission letter to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly)
-- Makes a final decision after receiving a recommendation from the Associate Editor and reviewers (see below)
+The Section Lead Editor:
+- Passes the article to an appropriate Associate Editor to manage the review process (unless he or she decides to manage the review process directly). Usually, this will be the Associate Editor who handled the presubmission letter; if not, the Section Lead editor also will communicate the approved presubmission letter.
+- Usually will delegate the final decision to the Associate Editor. If the Section Lead Editor anticipates their input may be necessary, they will notify the Associate Editor at the time of them anuscript submission, or if not, as soon as they become aware additional input may be necessary. They may overrule the Associate Editor after acceptance, but this should only be done in exceptional cases, for example, fraud or plagarism. If there are important scientific changes that need to be made in the article that were not caught by Associate Editors previous to acceptance, these can be handled by an unofficial request for revision before the paper is officially posted, or between ASAP posting and the issue publication.
```

Comment:
Suggest adding [...]"in exceptional cases [and after informing the Associate Editor]"

Reply:
Done.